infrastructure/templates/public-api/main.bicep

```diff
@@ -1,290 +1,206 @@
-
-@description('Storage : Name of the root fileshare folder name')
-@minLength(3)
-@maxLength(63)
-param fileShareName string = 'data'
-
-@description('Storage : Type of the file share quota')
+@description('Storage : Size of the file share in GB.')
 param fileShareQuota int = 1
 
-//PostgreSQL Database Params -------------------------------------------------------------------
-@description('Database : PostgreSQL server Name')
-param dbServerName string = 'publicapi-data'
-
-@description('Database : administrator login name')
+// PostgreSQL Database Params ------------------------------------------------------------------
+@description('Database : administrator login name.')
 @minLength(0)
-param dbAdminName string
+param postgreSqlAdminName string
 
-@description('Database : administrator password')
+@description('Database : administrator password.')
 @minLength(8)
 @secure()
-param dbAdminPassword string
+param postgreSqlAdminPassword string
 
-@description('Database : Azure Database for PostgreSQL sku name ')
+@description('Database : Azure Database for PostgreSQL sku name.')
 @allowed([
   'Standard_B1ms'
   'Standard_D4ads_v5'
   'Standard_E4ads_v5'
 ])
-param dbSkuName string = 'Standard_B1ms'
-
-@description('Database : Azure Database for PostgreSQL Storage Size in GB')
-param dbStorageSizeGB int = 32
-
-@description('Database : Azure Database for PostgreSQL Autogrow setting')
-param dbAutoGrowStatus string = 'Disabled'
-
-//Container Registry Params ----------------------------------------------------------------
-@minLength(5)
-@maxLength(50)
-@description('Registry : Name of the azure container registry (must be globally unique)')
-param containerRegistryName string
+param postgreSqlSkuName string = 'Standard_B1ms'
 
-@description('Deploy the Container Registry if you are not using an existing registry')
-param deployRegistry bool
+@description('Database : Azure Database for PostgreSQL Storage Size in GB.')
+param postgreSqlStorageSizeGB int = 32
 
-//Container App Params -------------------------------------------------------------------
-@minLength(2)
-@maxLength(32)
-@description('Specifies the name of the container app.')
-param containerAppName string
+@description('Database : Azure Database for PostgreSQL Autogrow setting.')
+param postgreSqlAutoGrowStatus string = 'Disabled'
 
-@description('Specifies the name of the container app environment.')
-param containerAppEnvName string = 'publicapi'
+@description('Database : Firewall rules.')
+param postgreSqlFirewallRules array = []
 
-@description('Specifies the name of the log analytics workspace.')
-param containerAppLogAnalyticsName string = 'publicapi'
+@description('Container App : Select if you want to use a public dummy image to start the container app.')
+param useDummyImage bool = true
 
-@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
-param containerAppImageName string
+param now string = utcNow('u')
```

Comment:
Hey, so yes this does need to be a param.  I originally tried to make the entire `tagValues` object a `var`, expecting that this value would just fall in line with that as well, but it seems that this one part of it needs to be dynamic in terms of being a param.  Presumably it needs to be injected in dynamically by the build process rather than being resolved by executing any code in the Bicep template itself.

I agree that it could use a rename, so `dateProvisioned` sounds good to me.